src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practice-setup.mdx

```diff
@@ -0,0 +1,152 @@
+
+Creating workloads unlocks other valuable features in New Relic, such as [errors inbox](/docs/errors-inbox/errors-inbox). With errors inbox, your teams can proactively detect, triage, and take action on all errors before they impact your customers. Errors are intelligently grouped to cut down on noise and ensure that critical errors are detected quickly and efficiently. This allows your team to resolve errors from across your stack, including errors from APM, browser (RUM), mobile, serverless (AWS Lambda) data, and OpenTelemetry, displayed on one screen. 
+
+As OpenTelemetry is implemented for your application services, we recommend using errors inbox to proactively review and remediate the most common errors before they impact end users. 
+
+## 8. Build dashboards for custom visualizations [#dashboards]
+
+New Relic provides a number of [out-of-the-box views of your OpenTelemetry data](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-view-your-data) to help get you started. As your teams start using the data for troubleshooting and identifying issues proactively, you may discover a need for custom visualizations. You can create custom dashboards for this purpose. 
+
+To learn the benefits of creating custom dashboards, and how to get started with them, see [Dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+## 9. Add context with custom instrumentation [#custom-instrumentation]
+
+Whether starting with automatic or manual instrumentation, it will often make sense to add additional instrumentation over time as needs for additional data arise while triaging issues. For example: 
+
+* Attributes can be added to spans to provide additional context about the request, which can be helpful during troubleshooting to understand the business impact of the issue (for example, adding the tenant ID to a span, to assist with troubleshooting a multi-tenant application). 
+* Additional nested spans can be captured, providing more detail for troubleshooting a particular aspect of the application more deeply. 
+* Custom metrics can be captured, from either a technical perspective (for example, tracking the number of cache hits and misses) or a business perspective (for example, tracking the average number of items in the cart during checkout). 
+
+The OpenTelemetry docs provide guidance and examples of these and more on a per-language basis. For example, when using Java you can use the following docs:
+
+* [Creating nested spans](https://opentelemetry.io/docs/instrumentation/java/manual/#create-nested-spans) 
+* [Span attributes](https://opentelemetry.io/docs/instrumentation/java/manual/#span-attributes)
+* [Metrics](https://opentelemetry.io/docs/instrumentation/java/manual/#metrics)
+
+We also have some examples that illustrate OpenTelemetry manual instrumentation using various languages.
```

Comment:
@zuluecho9 where are these examples? This seems like a teaser without any follow-up.